packages/design-system/stories/0-Text.stories.tsx

```diff
@@ -1,44 +1,125 @@
+
+    <Box>
+      <Text fontSize="xl" fontFamily="mono" color="white">
+        PΞTH
+      </Text>
+      <Text fontSize="md" color="white">
+        @peth
+      </Text>
+    </Box>
+
+    <Box>
+      <Text
+        fontSize="sm"
+        fontWeight="bold"
+        color="blueLight"
+        as="div"
+        textTransform="uppercase"
+      >
+        Role
+      </Text>
+      <Text fontSize="md" fontWeight="bold" color="white">
+        Community Lead
+      </Text>
+    </Box>
+
+    <Heading fontSize="sm" color="blueLight" as="div">
+      About me
+    </Heading>
+
+    <Box>
+      <Heading fontSize="2xl" color="white">
+        Peter Jursic
+      </Heading>
+      <Text fontSize="md" color="white">
+        Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet
+        sint. Velit officia consequat duis enim velit mollit. Exercitation
+        veniam consequat sunt nostrud amet.{' '}
+        <Button
+          variant="link"
+          size="sm"
+          color="cyanText"
+          fontWeight="normal"
+          verticalAlign="inherit"
+          onClick={() => {}}
+        >
+          Read more
+        </Button>
+      </Text>
+
+      <Link fontSize="sm" color="cyanText" href="https://metagame.wtf">
+        Go to external url
+      </Link>
+    </Box>
+
+    <Box>
+      <Text fontSize="lg" fontWeight="bold" color="white">
+        The Reformer
+      </Text>
+      <Text fontSize="sm" color="blueLight">
+        Principled, Purposeful, Self-Controlled, and Perfectionistic
+      </Text>
+    </Box>
+
+    <Box>
+      <Text fontSize="sm" fontWeight="bold" color="blueLight">
+        MetaGame Discord Bots
+      </Text>
+      <Text fontSize="sm" color="white">
+        Would be great to have the bot message every person promoted to MG
+        Player with “Welcome to the game…” (an onboarding guide, something like
+        this)
+      </Text>
+    </Box>
   </Stack>
 );
 
 export const Fonts = () => (
   <Stack spacing={3}>
-    <Text fontSize="lg" fontFamily="heading">
+    <Text fontSize="lg" fontFamily="heading" color="white">
       In love with MetaGame - Heading
     </Text>
-    <Text fontSize="lg" fontFamily="heading" fontWeight="bold">
+    <Text fontSize="lg" fontFamily="heading" fontWeight="bold" color="white">
       In love with MetaGame - Heading Bold
     </Text>
-    <Text fontSize="lg" fontFamily="body">
+    <Text fontSize="lg" fontFamily="body" color="white">
       In love with MetaGame - Body
     </Text>
-    <Text fontSize="lg" fontFamily="body" fontWeight="bold">
+    <Text fontSize="lg" fontFamily="body" fontWeight="bold" color="white">
       In love with MetaGame - Body Bold
     </Text>
-    <Text fontSize="lg" fontFamily="mono">
+    <Text fontSize="lg" fontFamily="mono" color="white">
       In love with MetaGame - Mono
     </Text>
-    <Text fontSize="lg" fontFamily="mono" fontWeight="bold">
+    <Text fontSize="lg" fontFamily="mono" fontWeight="bold" color="white">
```

Comment:
We shouldn't need to set these all as white if we just change the theme to Dark Mode. Can we wrap storybooks in the DarkMode theme? Feel free to do it in a separate PR and clean these up later if you prefer.